packages/calypso-build/README.md

```diff
@@ -123,19 +123,35 @@ module.exports = {
 The `default` preset has a `modules` option that specifies whether we want to transpile ESM `import` and `export` statements. Most common values are `false`, which keeps these statements intact and results in ES modules as output, and `'commonjs'`, which transpiles the module to the CommonJS format. See the [@babel/preset-env documentation](https://babeljs.io/docs/en/babel-preset-env#modules) for more details.
 
 ```js
-presets: [
-	[ '@automattic/calypso-build/babel/default', { modules: 'commonjs' } ]
-]
+presets: [ [ '@automattic/calypso-build/babel/default', { modules: 'commonjs' } ] ];
 ```
 
 Another way to set the `modules` option is to set the `MODULES` environment variable to `'esm'` (maps to `false`) or any other valid value. That's convenient for running Babel from command line, where specifying options for presets (`--presets=...`) is not supported.
 
+## Advanced Usage: Use own PostCSS Config
+
+You can also customize how PostCSS transforms your project's style files by adding a `postcss.config.js` to it.
+
+For example, the following `postcss.config.js` will use color definitions from `@automattic/calypso-color-schemes` by setting CSS variables, and add 'polyfills' for browsers like IE11:
+
+```js
+module.exports = () => ( {
+	plugins: {
+		'postcss-custom-properties': {
+			importFrom: [ require.resolve( '@automattic/calypso-color-schemes' ) ],
+			preserve: true,
```

Comment:
`preserve` can be removed from the docs example, too. I don't see any use case right now where we'd want to ever use `preserve: false`.

Reply:
54b4f03207